big-practice/src/js/login.js

```diff
@@ -46,17 +34,20 @@ const login = () => {
     .then((response) => response.json())
     .then((userList) => {
       if (userList.length === 0) {
-        warnMsg.innerHTML = MESSAGES.incorrectLoginAccount;
+        showErrorMessage(emailInput, MESSAGES.incorrectLoginAccount);
+        showErrorMessage(passwordInput, MESSAGES.incorrectLoginAccount);
         return;
       }
 
       // Non-admin account
       if (!userList[0].isAdmin) {
-        warnMsg.innerHTML = MESSAGES.notAdminAccount;
+        showErrorMessage(emailInput, MESSAGES.notAdminAccount);
```

Comment:
2 elements show the same message. I think should show one message only. Pls check around